tests/unit/remoteworker/test_remote_worker.py

```diff
@@ -5,8 +5,11 @@
 
 from os.path import join
 
-from unittest import TestCase
+from django.core.files.uploadedfile import SimpleUploadedFile
 
+from rest_framework.test import APITestCase
+
+from participants.models import Participant
```

Comment:
@takitsuse remove the new line change the imports on line 10 and 11 to 

```
from participants.models import Participant
from rest_framework.test import APITestCase
```

Imports should be in lexicographical order